test/extended/util/annotate/generated/zz_generated.annotations.go

```diff
@@ -2,6 +2,7 @@ package generated
 
 import (
 	"fmt"
+
```

Comment:
It looks like `make verify`  is unhappy about this

Reply:
it was my editor taking not needed initiatives